sanity/doctors/EstateDoctor.js

```diff
@@ -107,8 +107,14 @@ export class EstateDiagnosis extends Diagnosis {
     this.faultyEstates = faultyEstates
   }
 
-  getFaultyAssets() {
-    return this.faultyEstates
+  async getFaultyAssets() {
+    const faultyAssets = [...this.faultyEstates]
```

Comment:
why are we doing this? is `faultyParcels` a `Set` ?

Reply:
same as above